### DISCUSSION TITLE
Changes projected for API/certificate due to childs 5 to 11 years old with state 1/1?

### DISCUSSION THREAD

**Discussion by mge-medisoftware** [Digitaler-Impfnachweis/certification-apis#239] [Q&A]
Hello,

for healthy childs 5-11 years old only one vaccination is recommended in germany:

[Aktuelle Informationen zur COVID-19-Impfung](https://www.bundesgesundheitsministerium.de/coronavirus/faq-covid-19-impfung.html)

So ist state 1/1 possible/projected?

**Comment by Ein-Tim** [comment]
1/1 certificates are already possible, however, apps will assume that that's one vaccination after recovery.
